person.rb

```diff
@@ -19,6 +21,10 @@ def correct_name
     @name
   end
 
+  def add_rentals=(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] As requested by the previous reviewer. In the `add_rentals` method in the Person class, The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 

```ruby
 def add_rental(book, date)
    Rental.new(date, self, book)
end